library/ix-dev/community/homarr/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+name: homarr
+description: Homarr is a sleek, modern dashboard that puts all of your apps and services at your fingertips.
+annotations:
+  title: Homarr
+type: application
+version: 1.0.0
+apiVersion: v2
+appVersion: '1.1.3.2982'
```

Comment:
The `appVersion` seems to conflict with the tag version we are using in the docker image..is that expected?

Reply:
Missed that, thanks!